tests/data/config/malformed_config.yaml

```diff
@@ -0,0 +1,31 @@
+# The AdamSensors CSC reads voltages off of the ADAM 6024's six analog input channels, in
```

Comment:
Can you specify what is wrong with this configuration somewhere in a comment? it is not clear what is "malformed" about it.